src/bounce_solution/bounce_solution.cpp

```diff
@@ -855,6 +963,27 @@ void BounceSolution::CalculateCompletionTemp(const double &false_vac_frac)
   return;
 }
 
+void BounceSolution::CalculateReheatingTemp()
+{
+  if (alpha < 1)
+  {
+    Treh = GetTransitionTemp(); // no supercooling
+  }
+  else // supercooling
+  {
+    if (vwall < vCJ)
+    {
+      std::stringstream ss;
+      ss << "\n\033[1;93mWARNING: Detected wall velocity outside detonation "
```

Comment:
What is with the \033[1;93m ?

Reply:
This is to make the warning appear in color, for better visibility (like we do with other output in the code).